### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by MarkVed17**
- Added Filters Component
- Added Filters functionality
  -  Sort Notes by Time
  - Filter Notes by Label
  - Filter Notes by Priority
- Added Priorities Tag to a Notes (feature)
- Added Labels Screen
- Added Home Screen

### CODE REVIEW

File: src/frontend/components/Filters/Filters.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  text-decoration: underline;
+  text-align: center;
+  cursor: pointer;
+  color: var(--base-gray-color);
+}
+
+.filter-labels,
+.filter-priority {
+  /* border-right: 1px solid black; */
+  /* padding-right: 1rem; */
```
Comment (horsemaker) [inline src/frontend/components/Filters/Filters.css]: If you don't need these comments, do remove them.

**Review by horsemaker — APPROVED** [review]
Looks good to go! 👌

File: src/frontend/screens/LabelsScreen/LabelsScreen.js [hunk 2]
```diff
@@ -1,16 +1,71 @@
...
+      )
+    );
+  const labelTwoNotes = notes
+    .filter(
+      (note) =>
+        note.label === "Label 2" &&
+        note.isPinned === true &&
+        note.trash === false
+    )
+    .concat(
```
Comment (t007rushi) [inline src/frontend/screens/LabelsScreen/LabelsScreen.js:30]: no need to check explicitly just write note.trash  for boolean values

**Review by t007rushi — APPROVED** [review]
couldn't able to navigate to /notes path

File: src/frontend/components/Filters/Filters.css (hunk 1 above)
Comment (MarkVed17) [reply src/frontend/components/Filters/Filters.css]: Done.

**Comment by MarkVed17** [comment]
> couldn't able to navigate to /notes path

good that you noticed. Fixed!

**Pull request merged**